gfsphysics/GFS_layer/GFS_typedefs.F90

```diff
@@ -6112,6 +6118,11 @@ subroutine interstitial_create (Interstitial, IM, Model)
     allocate (Interstitial%dudt_mtb        (IM,Model%levs))
     allocate (Interstitial%dudt_ogw        (IM,Model%levs))
     allocate (Interstitial%dudt_tms        (IM,Model%levs))
+!-- GSD drag suite
```

Comment:
If these are only used for the GSD drag suite, then please allocate them only if the drag suite is used.